### ISSUE TITLE
Can't copy fernet key

### ISSUE THREAD

**Issue by deimosfr** [open-io/ansible-role-openio-keystone#4]
Hi,

Running on CentOS 7.4, I'm encountering issues to copy the fernet key.

If security restriction do not allow to log as root on servers, there is an issue to copy the key:

```
fatal: [oios01e02-par.bla.preprod -> oios01e01-par.bla.preprod]: FAILED! => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --archive --rsh=/usr/bin/ssh -S none -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null --out-format=<<CHANGED>>%i %n%L /etc/keystone/fernet-keys/ oios01e02-par.bla.preprod:/etc/keystone/fernet-keys/", "msg": "Warning: Permanently added 'oios01e02-par.bla.preprod,10.60.32.39' (ECDSA) to the list of known hosts.\r\nrsync: change_dir \"/etc/keystone/fernet-keys\" failed: Permission denied (13)\nrsync: ERROR: cannot stat destination \"/etc/keystone/fernet-keys/\": Permission denied (13)\nrsync error: errors selecting input/output files, dirs (code 3) at main.c(565) [Receiver=3.0.9]\nrsync: connection unexpectedly closed (9 bytes received so far) [sender]\nrsync error: error in rsync protocol data stream (code 12) at io.c(605) [sender=3.0.9]\n", "rc": 12}
```
If I'm looking at /etc/keystone folder, permissions are set to root:keystone with 0750. What I suggest is to grab the key locally, then send it to other hosts.

Note: I don't think changing /etc/keystone is a good alternative

If you're interested by a PR, please let me know. Otherwise, what do you propose?

Thanks

**Comment by deimosfr** [comment]
Here is the proposed fix #5

**Comment by racciari** [comment]
Thank you @deimosfr for your contribution !
